app/assets/javascripts/modules/map/leaflet-plugins/granule_layer.js.coffee

```diff
@@ -171,6 +171,19 @@ ns.GranuleLayer = do (L
       matched = false
       for optionSet in @multiOptions when @_matches(granule, optionSet.match)
         matched = true
+
+        # set resolution to {projection}_resolution if it exists
+        oldResolution = optionSet.resolution
+
+        if this._map.projection == 'geo'
```

Comment:
`newResolution = optionSet.geo_resolution if this._map.projection == 'geo'` makes it more uniform with line 180 and 181 :)